### ISSUE TITLE
Infinite loop when the -c parameter of the SimAI_simulator command is empty

### ISSUE THREAD

**Issue by dageita** [aliyun/SimAI#17]
I noticed that the parameter -c is required in the new version v1.0, and tutorial has not been updated yet. Using SimAI_simulator without -c will result in an infinite loop, I think this is due to this code segment:
```
bool ReadConf(string network_topo,string network_conf) {

    std::ifstream conf;
    conf.open(network_conf);
    topology_file = network_topo;
    while (!conf.eof()) {
      std::string key;
      conf >> key;
```
Should exception handler added?

**Comment by Huoyuan100861** [comment]
Yes, you need to add the `-c` flag to specify the configuration file, which is `astra-sim-alibabacloud/inputs/config/SimAI.conf`. 
The hanging issue is due to slight changes in the format of workload parsing in the new SimAI-Analytical (mainly to support VPP). You can try example/workload_analytical.txt.
I will update this part accordingly later.
